pages/common/helpers/index.js

```diff
@@ -0,0 +1,10 @@
+const getInspectors = req =>
+  req.api(`/search/profiles`)
```

Comment:
This should have a specific endpoint created to get the list of ASRU users. Using search will either only return a subset of the data (because paginated) or return _all_ the data, which is worse (because 27,000 people in prod)

Reply:
done.